custom/icds_reports/ucr/data_sources/child_health_cases_monthly_tableau2.json

```diff
@@ -1605,29 +1605,13 @@
         "column_id": "cf_in_month",
         "display_name": null,
         "filter": {
-          "type": "and",
-          "filters": [
-            {
-              "type": "boolean_expression",
-              "operator": "eq",
-              "property_value": "yes",
-              "expression": {
-                "type": "nested",
-                "argument_expression": {
-                  "type": "named",
-                  "name": "get_latest_cf_repeat"
-                },
-                "value_expression": {
-                  "type": "property_name",
-                  "property_name": "comp_feeding"
-                }
-              }
-            },
-            {
-              "name": "cf_eligible",
-              "type": "named"
-            }
-          ]
+          "type": "boolean_expression",
+          "operator": "eq",
+          "property_value": 0,
+          "expression": {
+            "type": "constant",
+            "constant": 1
+          }

```

Comment:
Huh, interesting idea, guessing this is to avoid a migration?

Reply:
Yep. There's a lot of `get_case_forms` in this data source. Once they're all gone, I plan to go back and either totally remove this data source and use others, or remove these columns in a new data source and copy data over